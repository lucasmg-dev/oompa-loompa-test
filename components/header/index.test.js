import '@testing-library/jest-dom'

import { Header } from './index'
import { render } from '@testing-library/react'

it('renders correctly', () => {
  const { asFragment } = render(<Header />)
  expect(asFragment()).toMatchSnapshot()
})
