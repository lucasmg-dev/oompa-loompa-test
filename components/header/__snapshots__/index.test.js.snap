// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<DocumentFragment>
  <header
    class="header"
  >
    <nav
      class="wrapper wrapper"
    >
      <a
        href="/"
      >
        <img
          class="logo"
          src=""
        />
        <span
          class="title"
        >
          Oompa Loompa's Crew
        </span>
      </a>
    </nav>
  </header>
</DocumentFragment>
`;
